demo/cache/cache-controller.js

```diff
@@ -1,8 +1,35 @@
-app.controller('CacheCtrl', function($scope, $rootScope, $http, FDP, Caching, Log) {    
+app.controller('CacheCtrl', function($scope, $rootScope, $http, FDP, Caching, Log, HttpEndpoint, File) {    
   $scope.isCachingStarted = false;
-  $scope.fairDataPoints=[{name: "RD connect FDP", url:"http://localhost:8084/fairdatapoint/fdp"}];
+  $scope.fairDataPoints = [];
   $scope.searchEngineUrl=null;
```

Comment:
Some genius wrote this line of code :-D, could you be kind enough to add space between the "="?, BTW, that genius is me ;-)

Reply:
Which line do you mean? 4?

Reply:
Yes, line no. 4